lib/ansible/modules/system/user.py

```diff
@@ -514,7 +526,13 @@ def modify_user_usermod(self):
 
         if self.expires:
             cmd.append('--expiredate')
-            cmd.append(time.strftime(self.DATE_FORMAT, self.expires))
+            try:
+                if self.clearexpires < 0:
+                    cmd.append('')
```

Comment:
This will most likely fail, since it sets an empty arg on --expiredate which will make usermod fail.